### ISSUE TITLE
This is great work!

### ISSUE THREAD

**Issue by benjamen** [dickreuter/betfair-bot-frontend#5]
I would like to see the same base concept for sports betting. Cricket for example has lot of value bet opportunities but without a bot it is difficult to take advantage of them.

**Comment by dickreuter** [comment]
Good idea. Do you know of any good data sources that could add value to
individual cricket games?

**Comment by benjamen** [comment]
Yes here you go:

ESPN Cricinfo: Website: https://www.espncricinfo.com/
Data: The most extensive repository of cricket data, covering international matches, domestic tournaments, and various player statistics.

2. Cricket Data API: Website: https://cricketdata.org/ (100 free hits per day)
Data: Free API providing live score updates, ball-by-ball data, current cricket scores, IPL, and fantasy cricket data.

3. Cricsheet: Website: https://cricsheet.org/
Data: Ball-by-ball data for various cricket matches, maintained by a passionate cricket fan. Offers downloadable CSV files for easy analysis.

4. Cricket Archive: Website: https://cricketarchive.com/ - PAID
Data: Extensive historical data on cricket matches and players, dating back to the 18th century. Requires paid subscription for full access.
